### PR TITLE
[Improvement]Remove useless code in test base

### PR DESCRIPTION
**Pull request by duanmeng**
### What changes were proposed in this pull request?
Remove the useless code that creat an extra coordinator server

### Why are the changes needed?
These codes are useless and kind of hardcode.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
ut and integration test

### CODE REVIEW

**Review by colinmjj — APPROVED** [review]
LGTM

File: integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java [hunk 1]
```diff
@@ -96,9 +96,6 @@ protected static ShuffleServerConf getShuffleServerConf() {
 
   protected static void createCoordinatorServer(CoordinatorConf coordinatorConf) throws Exception {
     coordinators.add(new CoordinatorServer(coordinatorConf));
-    coordinatorConf.setInteger("rss.rpc.server.port", COORDINATOR_PORT_2);
-    coordinatorConf.setInteger("rss.jetty.http.port", JETTY_PORT_2);
```
Comment (jerqi) [inline integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java:100]: we should need two coordinators

Comment (colinmjj) [reply integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java:100]: @jerqi yes, we should add case to test multiple coordinator.

Comment (jerqi) [reply integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java:100]: There are so many errors in ut's log that fail to send heartheat to secondary coordinator. @colinmjj @duanmeng If you don't have strong opinions on this, i will revert this patch.

Comment (colinmjj) [reply integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java:100]: @jerqi I think it's UT's problem, and ut should be fixed.
We shouldn't enable 2 coordinators with all situations.

Comment (jerqi) [reply integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java:100]: OK, Let's remove extra client.

**Pull request merged**